compiler-rt/cmake/config-ix.cmake

```diff
@@ -63,6 +63,16 @@ if (C_SUPPORTS_NODEFAULTLIBS_FLAG)
                         moldname mingwex msvcrt)
     list(APPEND CMAKE_REQUIRED_LIBRARIES ${MINGW_LIBRARIES})
   endif()
+  if (NOT TARGET unwind)
+    # Don't check for a library named unwind, if there's a target with that name within
+    # the same build.
+    check_library_exists(unwind _Unwind_RaiseException "" COMPILER_RT_HAS_LIBUNWIND)
```

Comment:
Can this use a different symbol? This symbol does not exist when SJLJ exceptions are enabled.

Reply:
Oh - good point. I guess we could check for `_Unwind_GetRegionStart` or `_Unwind_GetLanguageSpecificData` - both of those exist without any extra name mangling for both ehabi, sjlj, seh and regular dwarf.